### ISSUE TITLE
How do I choose which window to Miniview ?

### ISSUE THREAD

**Issue by stuaxo** [iamlemec/miniview#23]
I assumed this would always be Miniview the current Window, but every time I turn toggle it, it seems to be the same Window, am I missing something ?

**Comment by iamlemec** [comment]
Yep, so to switch what window is being shown, you can either use the scroll wheel (if you have one) on miniview, or hold shift and left/right click on miniview. The window order is sort of arbitrary and just mirrors whatever internal ordering the window manager uses. It probably has some relation to what order the windows were opened in though. Hope that helps!

**Comment by stuaxo** [comment]
Can I make a feature request that on opening it is whichever one is focused?

**Comment by iamlemec** [comment]
I think a lot of people rely on the target window being stable, so that you can quickly toggle miniview on and off. That said, it does make sense to default to the current window when either: (1) you're showing the miniview window for the first time that session; or (2) you're showing miniview again but had closed the target window while it was hidden.

Need to kind of test out that this logic makes sense. I'll put it on the todo list for the next release. Thanks for the feedback!

**Comment by stuaxo** [comment]
Having actually used it a little, I realised that "current Window" might not always make sense... it's worth testing this, vs something like "previous window"..  I haven't started using miniview much yet, so will see once I get used to it.
